doc/doc-site/docs/off-chain/python-sdk.md

```diff
@@ -1,3 +1,262 @@
+- `registry` ([move](https://github.com/econia-labs/econia/blob/main/src/move/econia/sources/registry.move)) ([python](https://github.com/econia-labs/econia/blob/main/src/python/sdk/econia_sdk/view/registry.py))
+- `user` ([move](https://github.com/econia-labs/econia/blob/main/src/move/econia/sources/user.move)) ([python](https://github.com/econia-labs/econia/blob/main/src/python/sdk/econia_sdk/view/user.py))
+- `resource_account` ([move](https://github.com/econia-labs/econia/blob/main/src/move/econia/sources/resource_account.move)) ([python](https://github.com/econia-labs/econia/blob/main/src/python/sdk/econia_sdk/view/resource_account.py))
+
+The corresponding Move code is well-commented and up-to-date, so it's helpful to have handy when working with the Python SDK. If the function you desire happens to not be supported by the Python SDK (or sadly isn't working), it's possible to use `econia_sdk.lib.EconiaViewer` to execute the appropriate `#[view]` Move function yourself. See the Python code linked above for examples of how an `EconiaViewer` instance is used. Note the return value of the `EconiaViewer` functions "[quacks](https://en.wikipedia.org/wiki/Duck_typing) like JSON" but every field-value is stringified.
+
+# Seconary Packages
+
+## `econia_sdk.lib`
+
+This package contains `EconiaClient`, which takes care of executing transactions pointed at `public entry` function targets, and `EconiaViewer` which takes care of access to off-chain callable `#[view]` function targets. These are not meant to be used without help from this Python SDK. Both are sufficiently capable to handle all possible functions to which they may apply, as long as those functions exist in the deployed Move contract code.
+
+## `econia_sdk.types`
+
+This package contains various enum types useful for parsing and referring to important values. Note that Move doesn't have Enum types so unlike most of the above, these do not map directly to Move. However, each value in each enum is associated with a constant that exists in the Move code.
+
+# Other Contents
+
+## `examples.trade`
+
+This is a script that makes use of both view functions and entry functions to perform a few scenarios in the exchange for the user. After set-up, it should be automatic (except for hitting enter to proceed the script). On display are things like registering a market, creating a market account and funding it, placing limit/market orders under various conditions as well as cancelling them. It's useful as a way to gain an understanding of what different fields/parameters mean and in general how to use the Python SDK. **Running/reading this example script and understanding what it does is recommended before anyone trades real money with the SDK.**
+
+### Running the Example Script ("Trading From Scratch")
+
+You're recommended to run a local Aptos node and faucet first; this prevents any rate-limitting issues from preventing your progress. Install the Aptos CLI & run your local node/faucet:
```

Comment:
Can you install the VS Code Cspell extension to VS Code and then "add limitting to workspace settings"?

Reply:
I don't see anything in the market place called Cspell

Reply:
Sorry, it's "Code Spell Checker"

Reply:
Looks like limitting is actually a typo, so now there is the correct spelling

Hence no need to add anything to the workspace dictionary, but the above extension should help with future typos